tests/phpunit/tests/rest-api/rest-themes-controller.php

```diff
@@ -471,6 +479,24 @@ public function test_theme_requires_wp() {
 		$this->assertSame( '5.3', $result[0]['requires_wp'] );
 	}
 
+	public function test_theme_is_block_theme() {
```

Comment:
Usually, the ticket number is provided in a docblock located above the function. Would you mind updating the PR accordingly? Thanks!

Reply:
+1 to the above, and this should also have an `@covers` annotation, so `@covers WP_REST_Themes_Controller::prepare_item_for_response` should be fine here.

See the proposed order/spacing of test docblocks in [this ticket](https://core.trac.wordpress.org/ticket/56070). Only `@ticket` and `@covers` are needed for this test method and also `test_theme_is_not_block_theme`.

Reply:
Done in https://github.com/WordPress/wordpress-develop/pull/4323/commits/91750e63e3edd38a613a40f7ef07b9e8dfc76228

Reply:
Thanks!